tests/conftest.py

```diff
@@ -13,6 +13,7 @@
 FIXTURES_FOLDER = TEST_FOLDER / 'fixtures'
 SPEC_FOLDER = TEST_FOLDER / "fakeapi"
 SPECS = ["swagger.yaml", "openapi.yaml"]
+OPENAPI_SPEC_ONLY = ["openapi.yaml"]
```

Comment:
IIRC "openapi" now encompasses both "v2" (fka swagger) and "v3".
so in general, it's good to clarify that you mean v3.